Makefile

```diff
@@ -131,6 +123,81 @@ create-env-file:
 	sudo echo "LOADTEST_CHILE_HOST=" | sudo tee -a /home/$(SSH_APP_USERNAME)/.env > /dev/null
 	sudo echo "NEWRELIC_APP_NAME=$$(hostname)" | sudo tee -a /home/$(SSH_APP_USERNAME)/.env > /dev/null
 	sudo echo "NEWRELIC_KEY=$(NEWRELIC_KEY)" | sudo tee -a /home/$(SSH_APP_USERNAME)/.env > /dev/null
+	sudo echo "LOADTEST_ALONE_MODE=$(LOADTEST_ALONE_MODE)" | sudo tee -a /home/$(SSH_APP_USERNAME)/.env > /dev/null
+
+## ARENA SERVER ONLY
+
+setup-caddy-arena:
+	sudo ufw allow 80
+	sudo ufw allow 443
+	sudo truncate -s 0 /etc/caddy/Caddyfile && \
+	echo "$$(hostname).championsofmirra.com {" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "  reverse_proxy localhost:4000" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "}" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	sudo sh -c 'echo "" >> /etc/caddy/Caddyfile'
+	echo "$$(hostname).prometheus.championsofmirra.com {" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "  basicauth {" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "    $$(PROMETHEUS_USER) $$(PROMETHEUS_PASSWORD)" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "  }" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "  reverse_proxy localhost:9568" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	echo "}" | sudo tee -a /etc/caddy/Caddyfile > /dev/null && \
+	sudo sh -c 'echo "" >> /etc/caddy/Caddyfile'
+	sudo systemctl restart caddy
+
+## CENTRAL SERVER ONLY
+
+debian-setup-grafana:
+	wget -q -O - https://apt.grafana.com/gpg.key | gpg --dearmor | sudo tee /etc/apt/keyrings/grafana.gpg > /dev/null
+	sudo apt-get update
+	sudo apt-get install grafana -y
+	sudo systemctl enable grafana-server
+	sudo systemctl start grafana-server
+
+setup-prometheus:
+	curl -s https://$(GATEWAY_URL)/api/arena_servers | \
+	jq -r '.servers[].url | "          - \(. | sub("^arena-"; "arena-") + ".prometheus.championsofmirra.com")"' > /tmp/targets.txt && \
+	echo "scrape_configs:" > /etc/prometheus/prometheus.yml && \
+	echo "  - job_name: 'arena_servers'" >> /etc/prometheus/prometheus.yml && \
+	echo "    scheme: https" >> /etc/prometheus/prometheus.yml && \
+	echo "    metrics_path: '/metrics'" >> /etc/prometheus/prometheus.yml && \
+	echo "    basic_auth:" >> /etc/prometheus/prometheus.yml && \
+	echo "      username: $$(PROMETHEUS_USER)" >> /etc/prometheus/prometheus.yml && \
+	echo "      password: $$(PROMETHEUS_PASSWORD)" >> /etc/prometheus/prometheus.yml && \
+	echo "    static_configs:" >> /etc/prometheus/prometheus.yml && \
+	echo "      - targets:" >> /etc/prometheus/prometheus.yml && \
+	cat /tmp/targets.txt >> /etc/prometheus/prometheus.yml && \
+	rm /tmp/targets.txt
+	sudo systemctl restart prometheus
+
+setup-central-caddy:
```

Comment:
aren't we missing the configurator reverse proxy? On central testing I see one

Reply:
Yup! I'll add it.